pyproject.toml

```diff
@@ -1,13 +1,13 @@
 [tool.poetry]
 name = "conversant"
-version = "0.2.2"
+version = "0.2.3"
 repository = "https://github.com/cohere-ai/sandbox-conversant-lib"
 description = "Conversational AI tooling"
 readme = "README.md"
 authors = ["Cohere ConvAI <convai@cohere.com>"]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.8,!=3.9.7"
```

Comment:
Because of [this](https://github.com/streamlit/streamlit/blob/0880740f482abd6a8f755192e3a94843a000f14c/lib/setup.py#L140), Streamlit doesn't work with 3.9.7